replace_with/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+// Julien Chien <jchien17@cmc.edu>
+
+pub fn replace_with<T, F: FnOnce(T) -> T>(t: &mut T, f: F) {
+  unsafe {
+    let mut ptr: *mut T = t;
+    *ptr = f(std::ptr::read(ptr))
```

Comment:
Yeah I'm curious too. 

```
$ rustc -V
rustc 1.12.1 (d4f39402a 2016-10-19)
$ cargo -V
cargo 0.13.0-nightly (109cb7c 2016-08-19)
```

And yes, "15" means "15 hours" (definitely not "15 minutes" :D )